rfcs/20201102-convert-verbose-into-benchdnn-problem/README.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Convert DNNL_VERBOSE output into benchdnn problems
+
+## Motivation
+
+There are several requests from users and developers to enable DNNL_VERBOSE
+output be "runnable" from benchdnn. Such requests mostly consider the following
+scenario: there is some integration of oneDNN into some framework, and there is
+some topology to run. If validation shows performance or correctness regression,
+it is very hard and/or time consuming to narrow down the specific guilty layer,
+and those users usually send a log file with DNNL_VERBOSE output, which is
+really massive. Manual conversion of these outputs into benchdnn problems takes
+enormous amount of time, since the topology usually consists of 100+ layers. It
+is just inconvenient not to have some tool that allows to perform such
+conversion very fast.
+
+## Tool Requirements
+
+* The tool should accept either a single line or a text file with lines.
+* The output should be fully benchdnn-compatible.
+* The tool should be easy to use.
+
+## Proposal
+
+### Option 1 (recommended) - Python script.
+
+Use the power of Python to create such tool. It will take the input and produce
+benchdnn batch files separated by drivers. Script advantages:
+* Flexibility in terms of features support.
+* Not working script will not affect a release.
+* It may have compatibility options between different options in benchdnn
+  drivers inside and will not depend on a specific version of the library and
+  benchdnn.
+
+### Option 2 - native benchdnn support.
+
+Implement such support as a standalone benchdnn driver. All pros of a Python
+script seems to be a cons of C++ solution:
+* Each feature should be tested through regular oneDNN validation cycles,
+  since it's a main validation tool.
+* Errors and bugs in code will likely lead to patch fixes for a release.
+* Versinong - earlier versions may lack of some suitable features.
+
+## Testing
+
+There is no clear vision about how the testing should like in either case. The
```

Comment:
```suggestion
There is no clear vision about how the testing should look like in either case. The
```

Reply:
I've re-wrote this section, so resolving as not relevant.